src/Uno.UI.RuntimeTests/Uno.UI.RuntimeTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="MSBuild.Sdk.Extras">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;xamarinios10;monoandroid80;monoandroid90;xamarinmac20;uap10.0.17763</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;xamarinios10;monoandroid90;xamarinmac20;uap10.0.17763</TargetFrameworks>
```

Comment:
Do not remove, this will be changed in a future PR.

Reply:
Added again

Reply:
Fixed the fact that is was not added again correctly, this change is not part of this PR anymore and will be changed in a future PR like @jeromelaban mentioned.